vignettes/rromeo.Rmd

```diff
@@ -15,54 +15,166 @@ vignette: >
+rr_publisher_country(country = "IR")
+```
+
+It is also possible to query publisher's information on a specific region or continent using `rr_publisher_continent()` (see the help page for the list of available regions):
+
+```{r query_publisher_continent}
+rr_publisher_continent(continent = "Australasia")
+```
+
+### Query by RoMEO colour
+
+RoMEO assigns a colour depending on the different policies of publishers.
+
+  | RoMEO colour | Archiving policy                                        |
+  |:-------------|:--------------------------------------------------------|
+  | `green`      | can archive preprint, postprint and publisher's version |
+  | `blue`       | can archive postprint **or** publisher's version        |
+  | `yellow`     | can archive preprint                                    |
+  | `white`      | archiving not formally supported                        |
+  
+(Table taken from <http://www.sherpa.ac.uk/romeo/definitions.php#colours>)
+
+You can query journals using this classification with the function `rr_romeo_colour()`:
+
+```{r query_romeo_colour, eval = FALSE}
+rr_romeo_colour(romeo_colour = "green")
+```
+
+In this example vignette we do not run the query because it can run for quite long as it returns the policies of all publishers of the given colour (you can see the numbers of publishers in each category in the following web page <http://www.sherpa.ac.uk/romeo/statistics.php?la=en&fIDnum=|&mode=simple>).
+
+
+## Setting up an API key
+
+SHERPA/RoMEO lets you make 500 queries per day per IP address for free. If you get past this limit you will get the following error:
+```
+You have exceeded the free use limit of 500 requests per day. To go beyond this limit you should register for a free API key available at http://www.sherpa.ac.uk/romeo/apiregistry.php
+```
+We encourage you to register for a **free** API key at the above-mentioned address.
+
+To provide your API key to `rromeo` you can:
+1. provide it as a character string as the `key` arguments of `rromeo` functions as `rr_*(..., key = "my_key_as_a_string")`, we **do not recommend** this method as your API key will be available from your history;
+1. you can define the variable `SHERPAROMEO_KEY` in an `.Renviron` file in your working directory, the file should contain the following line
+  `SHERPAROMEO_KEY=my_key_without_quotes`;
+1. you can also define the variable `SHERPAROMEO_KEY` in an `.Rprofile` file in your working directory, the file should contain the following line
+  `SHERPAROMEO_KEY="my_key_with_quotes"`.
+
+## Visualization Example
 
-This tool is useful in bibliometric studies. For example, we can now have a 
-visual overview of the policies of journals in a given field:
+`rromeo` can be quite useful in bibliometric studies to report on archival policies. For example, we can now have a quick visual overview of the policies of journals in a given field from the results obtained in a query:
 
 ```{r bar_graph}
 library("ggplot2")
```

Comment:
This does not match the style of other `library()` calls. Please remove the quotes